ratpack-core/src/main/java/ratpack/server/internal/ServerRegistry.java

```diff
@@ -116,6 +125,7 @@ public static Registry buildBaseRegistry(RatpackServer ratpackServer, Imposition
         .add(Clock.class, Clock.systemDefaultZone())
         .add(RatpackServer.class, ratpackServer)
         .add(ObjectMapper.class, new ObjectMapper())
+        .add(AddressResolverGroup.class, new DnsAddressResolverGroup(resolverBuilder))
```

Comment:
Users can switch back to the default JVM implementation by doing:
```
.add(AddressResolverGroup.class, DefaultAddressResolverGroup.INSTANCE)
```
in the user registry.